<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFRDRTableAxisSettingsWidget</class>
 <widget class="QWidget" name="QFRDRTableAxisSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>label:</string>
     </property>
     <property name="buddy">
      <cstring>edtXLabel</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="edtXLabel"/>
     </item>
     <item>
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>position:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="JKQTPlabelPositionComboBox" name="cmbXLabelPosition"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>range:</string>
     </property>
     <property name="buddy">
      <cstring>edtXMin</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="chkLogX">
       <property name="text">
        <string>log-scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFDoubleEdit" name="edtXMin"/>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFDoubleEdit" name="edtXMax"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnAutoscaleX">
       <property name="text">
        <string>autoscale</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_12">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ticks/labels:</string>
     </property>
     <property name="buddy">
      <cstring>cmbXMode1</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,0">
     <item row="1" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>labels:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="chkXAutoTicks">
         <property name="text">
          <string>min. #ticks / #minor ticks:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QSpinBox" name="spinXAxisMinTicks">
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinXAxisMinorTicks">
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>tick spacing:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>label rotation:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="spinLabelAngel">
         <property name="suffix">
          <string> Â°</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-90.000000000000000</double>
         </property>
         <property name="maximum">
          <double>90.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>15.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QFDoubleEdit" name="edtXTickSpacing">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="JKQTPCAlabelTypeComboBox" name="cmbXLabel"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>label digits:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinXAxisDigits">
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="1">
      <widget class="QLabel" name="label_50">
       <property name="text">
        <string>major ticks</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="spinXTickMinWidth">
       <property name="suffix">
        <string> pt</string>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_47">
       <property name="text">
        <string>width:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="spinXILength">
       <property name="suffix">
        <string> pt</string>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_52">
       <property name="text">
        <string>outside length:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="spinXMinILength">
       <property name="suffix">
        <string> pt</string>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_51">
       <property name="text">
        <string>minor ticks</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="spinXOLength">
       <property name="suffix">
        <string> pt</string>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_46">
       <property name="text">
        <string>inside length:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="spinXMinOLength">
       <property name="suffix">
        <string> pt</string>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="spinXTickWidth">
       <property name="suffix">
        <string> pt</string>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_31">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>properties:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>width:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="chkX0Axis">
       <property name="text">
        <string>show 0-axis</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkXInverted">
       <property name="text">
        <string>invert axis</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="spin0AxisWidth">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> pt</string>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_14">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ticks sizes:</string>
     </property>
     <property name="buddy">
      <cstring>cmbXMode1</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_15">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>named ticks:</string>
     </property>
     <property name="buddy">
      <cstring>cmbXMode1</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>values:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>labels:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFEnhancedComboBox" name="cmbNamedTicksValues"/>
     </item>
     <item row="1" column="1">
      <widget class="QFEnhancedComboBox" name="cmbNamedTicksNames"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_16">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>modes:</string>
     </property>
     <property name="buddy">
      <cstring>cmbXMode1</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0,1">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>axis 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="JKQTPCAdrawModeComboBox" name="cmbXMode1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>axis 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="JKQTPCAdrawModeComboBox" name="cmbXMode2"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_17">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>axis line widths:</string>
     </property>
     <property name="buddy">
      <cstring>cmbXMode1</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
     <item>
      <widget class="QDoubleSpinBox" name="spinAxis1Width">
       <property name="suffix">
        <string> pt</string>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actAddGraph">
   <property name="icon">
    <iconset resource="qfrdrtable.qrc">
     <normaloff>:/table/plot_add.png</normaloff>:/table/plot_add.png</iconset>
   </property>
   <property name="text">
    <string>add graph</string>
   </property>
  </action>
  <action name="actDeleteGraph">
   <property name="icon">
    <iconset resource="qfrdrtable.qrc">
     <normaloff>:/table/plot_delete.png</normaloff>:/table/plot_delete.png</iconset>
   </property>
   <property name="text">
    <string>delete graph</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qfdoubleedit.h</header>
   <slots>
    <signal>valueChanged(double)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>JKQTPCAlabelTypeComboBox</class>
   <extends>QComboBox</extends>
   <header>jkqtptools.h</header>
  </customwidget>
  <customwidget>
   <class>QFEnhancedComboBox</class>
   <extends>QComboBox</extends>
   <header>qfenhancedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>JKQTPCAdrawModeComboBox</class>
   <extends>QComboBox</extends>
   <header>jkqtptools.h</header>
  </customwidget>
  <customwidget>
   <class>JKQTPlabelPositionComboBox</class>
   <extends>QComboBox</extends>
   <header>jkqtptools.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtXLabel</tabstop>
  <tabstop>cmbXLabelPosition</tabstop>
  <tabstop>chkLogX</tabstop>
  <tabstop>edtXMin</tabstop>
  <tabstop>edtXMax</tabstop>
  <tabstop>btnAutoscaleX</tabstop>
 </tabstops>
 <resources>
  <include location="qfrdrtable.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chkXAutoTicks</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinXAxisMinTicks</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkXAutoTicks</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinXAxisMinorTicks</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkXAutoTicks</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkXAutoTicks</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtXTickSpacing</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkX0Axis</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkX0Axis</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin0AxisWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>graphDataChanged()</slot>
  <slot>plotDataChanged()</slot>
  <slot>plotColorChanged()</slot>
 </slots>
</ui>
